topostats/validation.py

```diff
@@ -563,6 +628,20 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                 "figure",
                 error="Invalid value in config for 'dpi', valid values are 'figure' or > 0.",
             ),
+            "module": Or(
```

Comment:
I'm not sure this is correct but may easily have misunderstood.

Doesn't the `module` field within the `plotting_dictionary.yaml` indicate what module the image belongs to?

If so the `inital_quadratic_removal` is an image from the `Filters()` stage of processing so should this not just be checking that the value is `filters` and nothing else, as we wouldn't want the image to be plotted as part of the `nodestats` set of images.

Same applies across all other additions to `topostats/validation.py`.

Reply:
Makes sense to me. I previously thought it was to check across the possible values, but this makes more sense. I wonder if this might be useful if someone want's to turn on/off plots this way - but this could just be done by switching the core_set field to True.

I've set it to just the module now :)